content/en-us/designs/protocol-translation/_index.md

```diff
@@ -50,7 +50,7 @@ Aggregation or batching of data is not a part of this design as it is discussed
 The Gateway design contains [considerations]({{ <ref "/designs/gateway#what-approach-should-be-used-when-storing-messages-for-later-delivery" >}}) about how protocol translation logic can be extended with mechanisms that batch results or compute aggregations over them.
 
 #### How should protocol translation process translated data when the network to the server is unavailable?
-The Gateway design contains some [considerations](/designs/gateway#how-should-the-gateway-process-data-when-the-network-to-the-server-is-unavailable) for this scenario. 
+The Gateway design contains some [considerations]({{ <ref "/designs/gateway#how-should-the-gateway-process-data-when-the-network-to-the-server-is-unavailable" >}}) for this scenario. 
```

Comment:
As above, positioning of the caret (`< ref`) in relation to the opening braces.:

The format for "ref" needs to be `{{< ref "test"` instead of `{{ <ref "text"` at the beginning. The rest is correct.